sources/rs/serbia-districts.hjson

```diff
@@ -0,0 +1,142 @@
+{
+    versions: '>=1'
+    type: http
+    production: true
+    wikidata: Q648681
+    note: District of Serbia
+    data: https://sophox.org/regions/geojson.json
+    attribution: [
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+            type: id
+            name: iso_3166_2
+            desc: ISO-3166-2 identifier
+        }
+        {
+            type: property
+            name: label_en
+            desc: Subdivision name (English)
+        }
+        {
+            type: property
+            name: label_sr
+            desc: Subdivision name (Serbian)
+        }
+    ]
+    query: {
+        sparql:
+            '''
+            #defaultView:MapRegions
+            SELECT
+            (if(bound(?id2),?id2,?id) as ?id)
+            ?iso_3166_2 ?label_en ?label_sr
+            WHERE {
+              SERVICE <https://query.wikidata.org/sparql> {
+              # Using nested query to ensure there is only one ?id2 value
+              {SELECT
+              ?id
+              (SAMPLE(?id2) as ?id2)
+              (SAMPLE(?iso_3166_2) as ?iso_3166_2)
+              (SAMPLE(REPLACE(?label_en," District","")) as ?label_en)
```

Comment:
:chef kiss: I like replacing the qualifiers in the SPARQL query. I wish it could be done for more countries.

Reply:
yeah, I realized it was doable directly on the query to avoid an extra step 😄